sdks/python/apache_beam/ml/inference/sklearn_inference.py

```diff
@@ -56,6 +56,7 @@ def run_inference(
       return SklearnInferenceRunner._predict_np_array(batch, model)
     elif isinstance(batch[0], pandas.DataFrame):
       return SklearnInferenceRunner._predict_pandas_dataframe(batch, model)
+    raise ValueError('Unsupported data type.')
```

Comment:
```suggestion
    raise ValueError('Unsupported data type. Please use either numpy ndarray or pandas DataFrame')
```
And can we also add a test for this?